great_expectations/execution_engine/pandas_execution_engine.py

```diff
@@ -71,37 +72,35 @@
 
 
 class PandasExecutionEngine(ExecutionEngine):
-    """
-PandasExecutionEngine instantiates the great_expectations Expectations API as a subclass of a pandas.DataFrame.
-
-For the full API reference, please see :func:`Dataset <great_expectations.data_asset.dataset.Dataset>`
-
-Notes:
-    1. Samples and Subsets of PandaDataSet have ALL the expectations of the original \
-       data frame unless the user specifies the ``discard_subset_failing_expectations = True`` \
-       property on the original data frame.
-    2. Concatenations, joins, and merges of PandaDataSets contain NO expectations (since no autoinspection
-       is performed by default).
-
---ge-feature-maturity-info--
-
-    id: validation_engine_pandas
-    title: Validation Engine - Pandas
-    icon:
-    short_description: Use Pandas DataFrame to validate data
-    description: Use Pandas DataFrame to validate data
-    how_to_guide_url:
-    maturity: Production
-    maturity_details:
-        api_stability: Stable
-        implementation_completeness: Complete
-        unit_test_coverage: Complete
-        integration_infrastructure_test_coverage: N/A -> see relevant Datasource evaluation
-        documentation_completeness: Complete
-        bug_risk: Low
-        expectation_completeness: Complete
-
---ge-feature-maturity-info--
+    """PandasExecutionEngine instantiates the ExecutionEngine API to support computations using Pandas.
+
+    For the full API reference, please see :func:`Dataset <great_expectations.data_asset.dataset.Dataset>`
+
+    Notes:
+        1. Samples and Subsets of PandaDataSet have ALL the expectations of the original data frame unless the user
+           specifies the ``discard_subset_failing_expectations = True`` property on the original data frame.
+        2. Concatenations, joins, and merges of PandaDataSets contain NO expectations (since no autoinspection is
+           performed by default).
+
```

Comment:
Let's remove this too, looks like it is part of the v2 API (it references PandasDataSet)
```suggestion
    Notes:
        1. Samples and Subsets of PandaDataSet have ALL the expectations of the original data frame unless the user
           specifies the ``discard_subset_failing_expectations = True`` property on the original data frame.
        2. Concatenations, joins, and merges of PandaDataSets contain NO expectations (since no autoinspection is
           performed by default).

```